### DISCUSSION TITLE
v8.x.x documentation?

### DISCUSSION THREAD

**Discussion by smathson** [pmndrs/react-spring#2122] [Support] [answered]
The new site looks beautiful and the v9 API looks much simplified and improved but for those of us still cursed with running an old v8.x.x version in production, there doesn't seem to be any way to reference the old API anymore unless I'm missing it somewhere. The docs were never in the repo so time-travelling back to https://github.com/pmndrs/react-spring/tree/v8.0.27 doesn't help. Do they still exist somewhere for technical reference?

**Comment by joshuaellis** [comment, marked as answer]
Yep, you can see the original documentation repo - https://github.com/pmndrs/react-spring.io

Unfortunately it's not online however. But it's just a next app, so all the content is in the pages directory.

**Reply by smathson** [reply]
Great thank you! Looks like this is the commit that captures the docs before the v9 changes in case anyone else needs to find them: https://github.com/pmndrs/react-spring.io/tree/18477e146f2fa45d70da705c880b677062c98a01
